### PR TITLE
Feature/65 jsx in depth

### PR DESCRIPTION
**Pull request by DennisBoanini**
Note to Hacktoberfest 🎃 participants:

While we appreciate the enthusiasm, we are experiencing a high volume of drive-by pull requests (one-line changes, README tweaks, etc.). Please remember that hundreds of people are subscribed to this repo and will receive notifications for these PRs. Spam submissions will be closed and won't count towards your Hacktoberfest goals.

Please search for issues tagged [`good first issue`][gfi] or [`hacktoberfest`][hacktoberfest] to find things to work on.

You can also search [all of GitHub][all].

[gfi]: https://github.com/reactjs/reactjs.org/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"
[hacktoberfest]: https://github.com/reactjs/reactjs.org/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"
[all]: https://github.com/search?q=is%3Aissue+hacktoberfest

### CODE REVIEW

File: content/docs/jsx-in-depth.md [hunk 1]
```diff
@@ -330,7 +331,7 @@ You can mix together different types of children, so you can use string literals
 </div>
 ```
 
-A React component can also return an array of elements:
+Un componente React può anche tornare un array di element:
```
Comment (deblasis) [inline content/docs/jsx-in-depth.md:334]:
```suggestion
Un componente React può anche tornare un array di elementi:
```

File: content/docs/jsx-in-depth.md [hunk 2]
```diff
@@ -346,15 +347,15 @@ render() {
 
 ### JavaScript Expressions as Children {#javascript-expressions-as-children}
 
-You can pass any JavaScript expression as children, by enclosing it within `{}`. For example, these expressions are equivalent:
+É possibile passare una qualsiasi espressione JavaScript, come elemento figlio, semplicemente circondandola da `{}`. Ad esempio queste espressioni sono tutte equivalenti:
```
Comment (deblasis) [inline content/docs/jsx-in-depth.md:350]:
```suggestion
É possibile passare una qualsiasi espressione JavaScript, come elemento figlio, semplicemente circondandola con `{}`. Ad esempio queste espressioni sono tutte equivalenti:
```

File: content/docs/jsx-in-depth.md [hunk 3]
```diff
@@ -402,11 +403,11 @@ function ListOfTenThings() {
 }
 ```
 
-Children passed to a custom component can be anything, as long as that component transforms them into something React can understand before rendering. This usage is not common, but it works if you want to stretch what JSX is capable of.
+I figli passati ad un componente custom possono essere qualsiasi cosa, visto che poi il componente li trasforma in qualcosa che React conosce prima di essere renderizzati. Questo uso non è comune, ma funiona se vuoi estendere tutte le capacità di JSX.
```
Comment (deblasis) [inline content/docs/jsx-in-depth.md:406]:
```suggestion
I figli passati ad un componente custom possono essere qualsiasi cosa, visto che poi il componente li trasforma in qualcosa che React conosce prima di essere renderizzati. Questo uso non è comune, ma funziona se vuoi estendere tutte le capacità di JSX.
```

**Review by deblasis — CHANGES_REQUESTED** [review]
Ottimo lavoro, giusto un paio di typos

**Comment by DennisBoanini** [comment]
> 
> 
> Ottimo lavoro, giusto un paio di typos

Grazie mille @deblasis . Dovrei aver applicato tutte le suggestion.

**Review by deblasis — APPROVED** [review]

**Comment by deblasis** [comment]
Closes #65

**Pull request merged**